experiment/dash/dash.go

```diff
@@ -3,42 +3,132 @@ package dash
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
+	"time"
 
+	"github.com/montanaflynn/stats"
+	"github.com/neubot/dash/client"
+	neubotModel "github.com/neubot/dash/model"
 	"github.com/ooni/probe-engine/experiment"
 	"github.com/ooni/probe-engine/experiment/handler"
-	"github.com/ooni/probe-engine/experiment/mkevent"
-	"github.com/ooni/probe-engine/measurementkit"
 	"github.com/ooni/probe-engine/model"
 	"github.com/ooni/probe-engine/session"
 )
 
 const (
-	testName    = "dash"
-	testVersion = "0.7.0"
+	testName       = "dash"
+	testVersion    = "0.8.0"
+	defaultTimeout = 55 * time.Second
+	maxRuntime     = 15.0
 )
 
 // Config contains the experiment config.
-type Config struct {
-	// LogLevel is the MK log level. Empty implies "WARNING".
-	LogLevel string
+type Config struct{}
+
+type Simple struct {
+	Connect_latency   float64 `json:"connect_latency"`
+	Median_bitrate    int64   `json:"median_bitrate"`
+	Min_playout_delay float64 `json:"min_playout_delay"`
+}
+
+// TestKeys contains the test keys
+type TestKeys struct {
+	Simple `json:"simple"`
```

Comment:
Can we avoid using embedding here? Why embedding only that structure and not others? I think that's confusing.

Reply:
This is the structure that was in MK. Doesn't that change cause problems for those using the API? 
https://explorer.ooni.org/measurement/20190913T235806Z_AS266347_h70ebKc0sVX7gz4TusAIpjfWeQjzUS4j2ZB6SPdT7yfZtIMASu

Reply:
I just mean you should write `Simple Simple `json:"simple"`

Reply:
For this I had mentioned somewhere this line:
https://github.com/google/gopacket/blob/33810c487ced200c983df5c45e3ebd2adc404b71/layers/tcp.go#L21
If you think "double `Simple`" is better. I'll do it.

Reply:
Yes, please, I'd find a single `Simple` confusing. My understanding is that embedding is Go's way to inherit from another "class" and I don't want to give this impression here.